spec/models/concerns/teacher_concern_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+require 'io/console'
```

Comment:
@kriskelly unfortunately I think that got in there while I was debugging something, but it turned out to not be necessary, yet I forget to take it out. Took it out in the subsequent commits...